mite/__main__.py

```diff
@@ -67,6 +67,7 @@
     --message-processors=PROCESSORS         Classes to connect to the message bus for local testing [default: mite.logoutput:HttpStatsOutput,mite.logoutput:MsgOutput]
     --prettify-timestamps                   Reformat unix timestamps to human readable dates
     --journey-logging                       Log errors on a per journey basis
+    --max-errors-threshold=THRESHOLD        Set the maximum amount of error accepted before set exit stauts to 1 [default: 0]
```

Comment:
```suggestion
    --max-errors-threshold=THRESHOLD        Set the maximum amount of error accepted before setting exit status to 1 [default: 0]
```

Reply:
I would go with "number of errors" rather than "amount of error"